File: Java/프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.java
class Solution {
    public String solution(String[] my_strings, int[][] parts) {
        String answer = "";
        int m=0;
        int n=0;
        int minus[] = new int[parts.length];
        // for(int i=0; i<parts.length; i++){
        //     minus[i]=parts[i][n+1]-parts[i][n];
        // }
        for(int i=0; i<parts.length;i++){
            for(int j=parts[i][0] ; j<parts[i][1]+1;j++){
                answer+=my_strings[i].charAt(j);
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181911. 부분 문자열 이어 붙여 문자열 만들기/부분 문자열 이어 붙여 문자열 만들기.js
function solution(my_strings, parts) {
    var answer = '';
    for(let i=0; i<parts.length;i++){
        for(let j=parts[i][0] ; j<parts[i][1]+1;j++){
            answer+=my_strings[i].charAt(j);
        }
    }
    return answer;
}